log/plugins.go

```diff
@@ -1,22 +1,42 @@
 package log
 
 import (
+	"context"
+	"fmt"
+
 	"golang.org/x/exp/slog"
 
-	"go-micro.dev/v5/util/container"
+	"github.com/go-orb/go-orb/types"
+	"github.com/go-orb/go-orb/util/container"
 )
 
-type pluginHandler struct {
-	handler slog.Handler
-	level   slog.Level
+// Provider can be started and stopped.
+type Provider interface {
+	fmt.Stringer
```

Comment:
Looks nicer to explicitly define String method

Reply:
true

Reply:
I've changed the Method from "String()" to "Key()" that makes it clearer what it does.